ulid_test.go

```diff
@@ -399,6 +400,31 @@ func TestEntropy(t *testing.T) {
 	}
 }
 
+func TestEntropyRead(t *testing.T) {
+	t.Parallel()
+
+	prop := func(e [10]byte) bool {
+		flakyReader := iotest.HalfReader(bytes.NewReader(e[:]))
+
+		id, err := ulid.New(ulid.Now(), flakyReader)
+		if err != nil {
+			t.Fatalf("got err %v", err)
+		}
+
+		got, want := id.Entropy(), e[:]
+		eq := bytes.Equal(got, want)
+		if !eq {
+			t.Errorf("\n(!= %v\n    %v)", got, want)
+		}
+
+		return eq
+	}
+
+	if err := quick.Check(prop, &quick.Config{MaxCount: 50}); err != nil {
```

Comment:
You can definitely do more than 50 runs. I think the default is 100, when not configured explicitly.